src/components/dashboard/account/UserSettings.js

```diff
@@ -0,0 +1,125 @@
+import React, { Component } from 'react';
+
+class UserSettings extends Component {
+	componentDidMount
+	render() {
+		const inputClass = 'input-reset ba pa2 mb2 db';
+		const labelClass = 'fw4 lh-copy f6 pr3 pt2 nowrap lh-solid'
+		const btnClass = 'b ph3 pv2 input-reset ba b--black bg-transparent grow pointer f6';
+
+		const user = this.props.user;
+
+		return (
+			<main className="pa4 black-80">
+					<form className="measure center" onSubmit={this.props.handleSubmit.bind(this)}>
+							<legend className="f4 fw6">Update User Details</legend>
+							<div className="ba pa4 mt2 mb2">
+									<div className="mt3 flex flex-row">
```

Comment:
we should store duplicate classes as a `const`, that way we only need to change it once to change multiple.